central/scanner/handler.go

```diff
@@ -93,8 +93,10 @@ func generateFilesForScannerV1(params *apiparams.Scanner, clusterType storage.Cl
 		ClusterType: clusterType,
 		K8sConfig: &renderer.K8sConfig{
 			CommonConfig: renderer.CommonConfig{
-				ScannerImage:   stringutils.OrDefault(params.ScannerImage, flavor.ScannerImage()),
-				ScannerDBImage: stringutils.OrDefault(params.ScannerDBImage, flavor.ScannerDBImage()),
+				ScannerImage:     stringutils.OrDefault(params.ScannerImage, flavor.ScannerImage()),
+				ScannerDBImage:   stringutils.OrDefault(params.ScannerDBImage, flavor.ScannerDBImage()),
+				ScannerV4Image:   stringutils.OrDefault(params.ScannerV4Image, flavor.ScannerV4Image()),
```

Comment:
The operator at current state allows setting different images for Scanner V4 Indexer and Scanner V4 Matcher. 

Although those Image are the same currently, we might consider doing this as well in here.

Reply:
Discussed in slack. Not going to change that for now.

Reply:
Discussed offline: Will keep the assumption for now that indexer and matcher share the same image, might be revisited later. This assumption is also present in the Helm charts and since others might already be reviewing at this point I figured we shouldn't continue working on such enhancements which are serving a -- currently -- theoretical use-case.